src/layouts/MapLayout.tsx

```diff
@@ -15,7 +16,6 @@ interface OnSelectOutput {
 export const MAP_CONFIG = {
   minZoom: 11.5,
   maxZoom: 22,
-  defaultZoom: 14,
   defaultLatitude: 52.520952,
   defaultLongitude: 13.400033,
```

Comment:
I would suggest to use a slightly different Latitude. Since we have this little menu bar at the bottom of the page it looks like the initial mapView is cut / not centered. You can see this effect very good on mobile

**Proposal:** use a smaller Latitude value --> looks ok: 52.51

Otherwise very good :)